pkgs/development/tools/micronaut/default.nix

```
@@ -16,7 +16,8 @@ stdenv.mkDerivation rec {
     rm bin/mn.bat
     cp -r . $out
     wrapProgram $out/bin/mn \
-      --prefix JAVA_HOME : ${jdk} 
+      --prefix JAVA_HOME : ${jdk} \
```

```suggestion
      --set JAVA_HOME ${jdk} \
```
JAVA_HOME is one path, not multiple ones like PATH

---

Good catch. I will have to update this PR.